packages/esm-patient-attachments-app/src/attachments/attachment-thumbnail.component.tsx

```diff
@@ -13,7 +13,7 @@ export default function AttachmentThumbnail(props: AttachmentThumbnailProps) {
 function ImageThumbnail(props: ImageProps) {
   return (
     <div className={styles.imageThumbnail} role="button" tabIndex={0} onClick={props?.onClick}>
-      <img src={props.src} alt={props.title} style={props.style} />
+      <img src={props.src} style={props.style} />
```

Comment:
We'll still need to show some sort of `alt` text.

Reply:
Oh I see!
Since we are not using imageProps anywhere else I though to clean up the code by removing the title property.
What can be done on case of `alt` ? Make the title in image props?

Reply:
You want to have some sort of alt text to show in case the image doesn't get rendered for whatever reason. Also, could you show the image caption or description just above the date so things are aligned with the [design](https://zpl.io/bWG8yg1)?

![Uploading CleanShot 2024-01-30 at 12 .19.02@2x.png…]()

Reply:
cool!
I've made it back `alt` to image.title as we don't have description and changed the typography too
@denniskigen

Reply:
Excellent. Thanks, @Vijaykv5!